seed/static/seed/js/controllers/inventory_list_beta_controller.js

```diff
@@ -122,6 +122,20 @@ angular.module('BE.seed.controller.inventory_list_beta', [])
       $scope.column_filters = []
       $scope.column_sorts = []
 
+      // remove editing on the filter list input (ngTagsInput doesn't support readonly yet)
+      let findFiltersListAttempts = 0;
+      const findFiltersList = setInterval(() => {
```

Comment:
so this makes the filters input read only? not crazy about the setInterval, but looks like its our best option